code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
Thing is that doesn't remove it from the air_master, I don't believe, nor does it allow DestroyTurf() to get called. I realize it's an inconsistency, but having it clear the air_master and also DestroyTruf seems a bit more "foolproof" and "robust" so to speak.